.github/workflows/release.yml

```
@@ -50,9 +51,12 @@ jobs:
         run: |
           echo $CONDA/bin >> $GITHUB_PATH
       - name: constructor build
+        env:
+          CONDA_SOLVER: libmamba
         run: |
           conda info
-          conda install constructor -y
+          conda install conda-libmamba-solve
+          conda create -n demo --experimental-solver=libmamba --dry-run install constructor -y
```

```suggestion
          conda create -n demo --experimental-solver=libmamba --dry-run install constructor conda-libmamba-solver -y
```

Maybe we need it here as well, not sure.